trade-finance-manager-api/src/constants/date.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
i think we should move this to libs common, but not worry about complete commonisation of legacy code for this PR